tests/run-tests.sh

```diff
@@ -443,6 +443,14 @@ check_exit_code() {
     fi
 }
 
+# verify that yum is updated
```

Comment:
we are checking for updates on all the distros. pls change yum to package as "verify if package update works"